### ISSUE TITLE
Writing: Expand acronyms on their first use

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-quick-start#122]
How about just "Expand acronyms", or, better, "Provide expansions or definitions for acronyms"?

Rationale: WCAG requires for using the acronym (abbr) _element_ for *at least* the first occurrence but the requirement is actually more general - acronyms should be understandable at all times. There are other techniques that could be used, such as a glossary, as well.

You can reflect this in the description too:

[[
Ensure that acronyms are defined or expanded, to help users understand them. You can use the abbr element on the first occurrence of the acronym to provide an expansion for it. This does not apply to acronyms that have become proper names or vernacular, such as 'Abba', 'IKEA', and 'Laser'.
]]

**Comment by iadawn** [comment]
Closing this in light of discussion around #158
